contrib/adacut

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
```

Comment:
shouldn't this file be "adacut.py"?

Reply:
Ok for that. That's a hard to shake unix habit to hide exe extensions, but it will fail for windows users.